packages/core/scripts/register-contract-umip/1_Propose.js

```diff
@@ -0,0 +1,92 @@
+// This script generates and submits a transaction that registers a new price requester for the DVM. It can be
+// run on a local ganache fork of the main net or can be run directly on the main net to execute the upgrade
```

Comment:
Extremely minor nit: `main net` is inconsistent with typical spelling.

Reply:
Thanks! Fixed for all other UMIP scripts as well